packages/ui-components/src/components/DateRangePicker/useDateRangePicker.ts

```diff
@@ -31,24 +31,27 @@ const DEFAULT_GRANULARITY = GRANULARITIES.DAY;
  * @returns {*|string}
```

Comment:
Let's delete the doc blocks since we now have the type information which is better.